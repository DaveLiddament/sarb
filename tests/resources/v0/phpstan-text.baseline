{"historyAnalyser":"git","historyMarker":"0f078b7d445f661991605b91d8ddb9c7f796242d","resultsParser":"phpstan-text-tmp","analysisResults":[{"lineNumber":0,"fileName":"static-analysis-baseliner\/src\/Employee.php","type":"Class not found and could not be autoloaded.","message":"Class Demo\\Foo not found and could not be autoloaded.","fullDetails":"\/vagrant\/static-analysis-baseliner\/src\/Employee.php:0:Class Demo\\Foo not found and could not be autoloaded."},{"lineNumber":50,"fileName":"static-analysis-baseliner\/src\/Plugins\/PsalmTextResultsParser\/PsalmTextResultsParser.php","type":"Call to an undefined method","message":"Call to an undefined method DaveLiddament\\StaticAnalysisResultsBaseliner\\Plugins\\PsalmTextResultsParser\\PsalmTextResultsParser::processLine().","fullDetails":"\/vagrant\/static-analysis-baseliner\/src\/Plugins\/PsalmTextResultsParser\/PsalmTextResultsParser.php:50:Call to an undefined method DaveLiddament\\StaticAnalysisResultsBaseliner\\Plugins\\PsalmTextResultsParser\\PsalmTextResultsParser::processLine()."}]}