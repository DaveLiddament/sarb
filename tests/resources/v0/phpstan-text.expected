{
    "historyAnalyser": "git",
    "historyMarker": "0f078b7d445f661991605b91d8ddb9c7f796242d",
    "resultsParser": "phpstan-json",
    "analysisResults": [
        {
            "lineNumber": 0,
            "fileName": "static-analysis-baseliner\/src\/Employee.php",
            "type": "Class not found and could not be autoloaded.",
            "message": "Class Demo\\Foo not found and could not be autoloaded.",
            "severity": "error"
        },
        {
            "lineNumber": 50,
            "fileName": "static-analysis-baseliner\/src\/Plugins\/PsalmTextResultsParser\/PsalmTextResultsParser.php",
            "type": "Call to an undefined method",
            "message": "Call to an undefined method DaveLiddament\\StaticAnalysisResultsBaseliner\\Plugins\\PsalmTextResultsParser\\PsalmTextResultsParser::processLine().",
            "severity": "error"
        }
    ]
}
